oandm/sprint-suspicious-activity-and-data-exfiltration/sprint-suspicious-activity-and-data-exfiltration.md

```diff
@@ -0,0 +1,98 @@
+# How can I set up an alert for detecting suspicious activity and potential data exfiltration from logs using Logging Analytics?
+
+Duration: 15 minutes
+
+This lab will walk you through the steps to Create & verify Scheduled Detection rules alarms to detect **Suspicious Activity and Data Exfiltration**.
+
+This lab explains scheduled task detection rules based on numerical thresholds.
+
+1. Navigate to the logs explorer (Refer to [How to use Log Explorer for analytics and visualizations?](?lab=sprint-log-explorer-for-analytics-and-visualizations) for more info) and run this query:
+
+    ```MQL
+    <copy>'Log Source' = 'OCI Audit Logs' and Event = interactivelogin and Status = '400' and Message = 'InteractiveLogin failed' | distinct 'Client Host City', 'User Name'</copy>
+    ```
+
+2. Save the search:
+
+    1. Click on the **"Actions"** button and Choose **"Save as..."**:
+    ![Figure 1: Open the Saved Search save panel](images/open-saved-search-save-panel.png)
+
+    2. Fill the Saved Search creation form and click on **"save"**:
+      - Choose a compartment
+      - Fill the **name** field for the saved search (For the rest of this live lab, we will name it "Failed Login attempts to OCI tenancy")
+      - Feel free to add a **description** to your saved search
+    ![Figure 2: Save the "Log Explorer" Search](images/save-search-from-log-explorer.png)
+
+3. Create the Detection Rule (Refer to [How to create and trigger an ingest time detection rule?](?lab=create-scheduled-search-detection-rule) for more details):
+
+    1. Click on **"Actions"** and Select **"Create Detection Rule"**:
+    ![Figure 3: Open the Detection Rule's creation Panel](images/open-detection-rule-creation-panel.png)
+
+    2. Fill the form to create the Detection Rule and hit **"Save"**:
+
+      - **Rule name:** The name of your Detection Rule.
+      - **Metric Compartment:** Your metrics compartment.
+      - **Metric Namespace:** A friendly name for your metrics Namespace.
+      - **Metric Name:** A friendly name for your Metric, we will set it to *unsuccessful_logins* in this live lab.
+      - **Interval:** The interval after which the Detection Rule should be re-run, We will set it to 5 min in this live lab.
+
+      **P.S:** The metric namespace and metric name will be used to create the alarms in a later step in this live lab.
+
+      ![Figure 4: Create a Detection Rule from the Saved Search](images/create-detection-rule-from-saved-search.png)
+
+Your Detection Rule should be up and ready and you should see a notification similar to this one:
+
+![Figure 5: Detection Rule creation Confirmation Message](images/detection-rule-creation-confirmation-message.png)
+
+Trigger the Detection Rule and make its metrics available in the alarms service. The fastest way to achieve this is by signing in to your tenancy with a wrong password.
+
+Create an alarm and verify it:
```

Comment:
use `##` to create a secondary header format to differentiate this new set of steps

Reply:
Done!